.github/workflows/lib-ci-dev.yaml

```diff
@@ -169,15 +169,18 @@ jobs:
       with:
         python-version: '3.10'
 
+    - name: Install uv
+      uses: astral-sh/setup-uv@v6
+
     - name: install dependencies
       run: |
-        python -m pip install --upgrade pip
-        pip install -q toml
-        pip install -q https://github.com/qiime2/q2lint/archive/master.zip
-        pip install -q flake8
+        uv venv
+        uv pip install -q toml https://github.com/qiime2/q2lint/archive/master.zip
```

Comment:
Using [uv's pip interface](https://docs.astral.sh/uv/pip/) is only necessary here because q2lint doesn't have a pyproject.toml file, I think. Note that this isn't actually using pip.

> uv does not rely on or invoke pip. The pip interface is named as such to highlight its dedicated purpose of providing low-level commands that match pip's interface and to separate it from the rest of uv's commands which operate at a higher level of abstraction.